matrix-synapse-riot/terraform/matrix-server/dehydrated-hooks

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env bash
+
+# Define DO API token
+DO_TOKEN=TOKENSTRING
+DO_DOMAIN=DODOMAIN
+
+deploy_challenge() {
+    local DOMAIN="${1}" TOKEN_FILENAME="${2}" TOKEN_VALUE="${3}"
+
+    # This hook is called once for every domain that needs to be
+    # validated, including any alternative names you may have listed.
+    #
+    # Parameters:
+    # - DOMAIN
+    #   The domain name (CN or subject alternative name) being
+    #   validated.
+    # - TOKEN_FILENAME
+    #   The name of the file containing the token to be served for HTTP
+    #   validation. Should be served by your web server as
+    #   /.well-known/acme-challenge/${TOKEN_FILENAME}.
+    # - TOKEN_VALUE
+    #   The token value that needs to be served for validation. For DNS
+    #   validation, this is what you want to put in the _acme-challenge
+    #   TXT record. For HTTP validation it is the value that is expected
+    #   be found in the $TOKEN_FILENAME file.
+
+    # Simple example: Use nsupdate with local named
+    # printf 'server 127.0.0.1\nupdate add _acme-challenge.%s 300 IN TXT "%s"\nsend\n' "${DOMAIN}" "${TOKEN_VALUE}" | nsupdate -k /var/run/named/session.key
+    DO_HOSTNAME=$(echo $DOMAIN | sed -e "s/$DO_DOMAIN//" | sed 's/.$//')
+    echo "Issueing channenge for $DO_HOSTNAME Lets Encrypt domain is $DOMAIN Challenge is $TOKEN_VALUE"
```

Comment:
Spellings `Issuing` and `challenge`. There are a couple more spelling mistakes throughout the messages, please run a check.

Reply:
The format of the UI logging can use some improvement.